src/angular-app/bellows/core/offline/editor-data.service.ts

```diff
@@ -450,18 +451,38 @@ export class EditorDataService {
     return input.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'); // $& means the whole matche
   }
 
+  private normalizeDiacritics(input: string) {
```

Comment:
```suggestion
  private removeDiacritics(input: string) {
```
Suggest a function rename since the returned string actually has no diacritics, not just normalized to NFD.

Reply:
yep, good call!